### PR TITLE
Refactoring entirely Caticorn PCB

### PR DESCRIPTION
**Pull request by HorrorTroll**
## Description

Refactoring entirely Caticorn PCB

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by dunk2k** [comment]
As per PR Contributing guidelines, these changes should be targeted at `develop` branch

File: keyboards/horrortroll/caticorn/rev1/hotswap/info.json [hunk 1]
```diff
@@ -1,43 +1,45 @@
...
+        "console": false,
+        "command": false,
+        "nkro": true
+    },
     "community_layouts": ["tkl_f13_ansi"],
     "layouts": {
         "LAYOUT_tkl_f13_ansi": {
             "layout": [
                 {"matrix": [0, 0], "x": 0, "y": 0},
-
```
Comment (fauxpark) [inline keyboards/horrortroll/caticorn/rev1/hotswap/info.json:23]: Please keep the spacing here.

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Comment by HorrorTroll** [comment]
Thanks both of you guys for pointing me out some issues. I've just pushed new commit to resolved these problem, and re-target from `master` to `develop` branch. If you have time, feel free to checked.

**Comment by dunk2k** [comment]
@HorrorTroll 
> Would you be able to cross reference `tkl` entries in `/layouts/default` for matching layouts this PCB can accommodate? adding matching layouts to `"community_layouts"` and "`layouts"` in info.json.

**Comment by HorrorTroll** [comment]
Ummm, I've still not know what that mean?

Did you mean solder or hotswap PCB? If it the solder PCB, I've been added all the `TKL` entries from `/layouts/default`, because the PCB supported all of it.

While hotswap board is just only `tkl_f13_ansi` anyway. Thanks!
@dunk2k

**Comment by dunk2k** [comment]
> I've been added all the `TKL` entries from `/layouts/default`, because the PCB supported all of it.
> 
> While hotswap board is just only `tkl_f13_ansi` anyway. 

Thanks for clarifying 😊

**Review by fauxpark — APPROVED** [review]

File: keyboards/horrortroll/caticorn/rev1/hotswap/keymaps/default/keymap.c [hunk 2]
```diff
@@ -23,11 +23,9 @@
 
 enum layer_names {
     _BASE,
-    _FN,
+    _FN
```
Comment (drashna) [inline keyboards/horrortroll/caticorn/rev1/hotswap/keymaps/default/keymap.c:26]: This should be reverted. 
```suggestion
    _FN,
```
While not critical, clang-format will condense this to one line for the entire enum. The comma at the end prevents it from doing so.

File: keyboards/horrortroll/caticorn/rev1/hotswap/keymaps/via/keymap.c [hunk 3]
```diff
@@ -23,11 +23,9 @@
 
 enum layer_names {
     _BASE,
-    _FN,
+    _FN
```
Comment (drashna) [inline keyboards/horrortroll/caticorn/rev1/hotswap/keymaps/via/keymap.c:26]:
```suggestion
    _FN,
```

**Review by drashna — APPROVED** [review]
a couple of nits, but otherwise, LGTM.

**Comment by HorrorTroll** [comment]
Thanks, I've just resolved the suggested. If you have time, please take a look.

**Pull request merged**